### PR TITLE
feat: new minify option

### PR DESCRIPTION
**Pull request by Dimitris-Toulis**
Minify the generated manifest to save some bytes

### CODE REVIEW

**Comment by antfu** [comment]
I think we could simply have `minify` as the options name

File: src/html.ts [hunk 1]
```diff
@@ -3,7 +3,8 @@ import { FILE_MANIFEST, FILE_SW_REGISTER } from './constants'
 import { ResolvedVitePWAOptions } from './types'
 
 export function generateSWRegister(options: ResolvedVitePWAOptions) {
-  return `
+  return options.minify ? `"serviceWorker"in navigator&&window.addEventListener("load",()=>{navigator.serviceWorker.register("${join(options.basePath, options.filename)}",{scope:"${options.scope}"})});` : 
+`
```
Comment (antfu) [inline src/html.ts:7]: Don't think it's necessary here, can you omit this change?

Comment (Dimitris-Toulis) [reply src/html.ts]: Done!

**Comment by antfu** [comment]
Thanks

**Pull request merged**